### PR TITLE
ansible 2.10.6

### PR DESCRIPTION
**Pull request by okket**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by okket** [comment]
I followed the instructions found in the formula and https://github.com/Homebrew/homebrew-core/pull/50504#issuecomment-592743222 for the dependencies.

For more details see also the release announcement https://groups.google.com/g/ansible-announce/c/6_AmfDp2fCw

Esp. this:
```
The next major release of Ansible (3.0.0) is scheduled for February 16, 2021. 
```

File: Formula/ansible.rb [hunk 1]
```diff
@@ -187,6 +187,11 @@ class Ansible < Formula
     sha256 "5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b"
   end
 
+  resource "cryptography" do
+    url "https://files.pythonhosted.org/packages/b7/82/f7a4ddc1af185936c1e4fa000942ffa8fb2d98cff26b75afa7b3c63391c4/cryptography-3.3.1.tar.gz"
+    sha256 "7e177e4bea2de937a584b13645cab32f25e3d96fc0bc4a4cf99c27dc77682be6"
+  end
+
```
Comment (laserscout) [inline Formula/ansible.rb:194]: This resource is a duplicate from the setup_requires dependencies, extras for requests section

File: Formula/ansible.rb [hunk 2]
```diff
@@ -222,6 +227,11 @@ class Ansible < Formula
     sha256 "0b0e026b412a0ad096e753907559e4bdb180d9ba9f68dd9036164db4fdc4ad2e"
   end
 
+  resource "idna" do
+    url "https://files.pythonhosted.org/packages/ea/b7/e0e3c1c467636186c39925827be42f16fee389dc404ac29e930e9136be70/idna-2.10.tar.gz"
+    sha256 "b307872f855b18632ce0c21c5e45be78c0ea7ae4c15c828c20788b26921eb3f6"
+  end
+
```
Comment (laserscout) [inline Formula/ansible.rb:234]: ^^

File: Formula/ansible.rb [hunk 3]
```diff
@@ -377,6 +387,11 @@ class Ansible < Formula
     sha256 "defd50f72b65c5402ab2c573830a6978e5f202ad0d984793c8dde2c4152ebe04"
   end
 
+  resource "pbr" do
+    url "https://files.pythonhosted.org/packages/65/e2/8cb5e718a3a63e8c22677fde5e3d8d18d12a551a1bbd4557217e38a97ad0/pbr-5.5.1.tar.gz"
+    sha256 "5fad80b613c402d5b7df7bd84812548b2a61e9977387a80a5fc5c396492b13c9"
+  end
+
```
Comment (laserscout) [inline Formula/ansible.rb:394]: ^^

File: Formula/ansible.rb [hunk 4]
```diff
@@ -392,6 +407,11 @@ class Ansible < Formula
     sha256 "5c5d0a3b48ceee0b48485e0c26037c0acd7d29765ca3fbb5cb3831d347423220"
   end
 
+  resource "pyOpenSSL" do
+    url "https://files.pythonhosted.org/packages/98/cd/cbc9c152daba9b5de6094a185c66f1c6eb91c507f378bb7cad83d623ea88/pyOpenSSL-20.0.1.tar.gz"
+    sha256 "4c231c759543ba02560fcd2480c48dcec4dae34c9da7d3747c508227e0624b51"
+  end
+
```
Comment (laserscout) [inline Formula/ansible.rb:414]: ^^

File: Formula/ansible.rb [hunk 5]
```diff
@@ -473,6 +493,11 @@ class Ansible < Formula
     sha256 "dcf9060b03f07647c0a603408dc8b03f807f3b54a05c6e19eb14460256fac0cb"
   end
 
+  resource "pytz" do
+    url "https://files.pythonhosted.org/packages/70/44/404ec10dca553032900a65bcded8b8280cf7c64cc3b723324e2181bf93c9/pytz-2020.5.tar.gz"
+    sha256 "180befebb1927b16f6b57101720075a984c019ac16b1b7575673bea42c6c3da5"
+  end
+
```
Comment (laserscout) [inline Formula/ansible.rb:500]: ^^

File: Formula/ansible.rb [hunk 6]
```diff
@@ -488,16 +513,16 @@ class Ansible < Formula
...
   resource "requests-oauthlib" do
     url "https://files.pythonhosted.org/packages/23/eb/68fc8fa86e0f5789832f275c8289257d8dc44dbe93fce7ff819112b9df8f/requests-oauthlib-1.3.0.tar.gz"
     sha256 "b4261601a71fd721a8bd6d7aa1cc1d6a8a93b4a9f5e96626f8e4d91e8beeaa6a"
   end
 
+  resource "requests_ntlm" do
+    url "https://files.pythonhosted.org/packages/3e/02/6b31dfc8334caeea446a2ac3aea5b8e197710e0b8ad3c3035f7c79e792a8/requests_ntlm-1.1.0.tar.gz"
+    sha256 "9189c92e8c61ae91402a64b972c4802b2457ce6a799d658256ebf084d5c7eb71"
+  end
+
```
Comment (laserscout) [inline Formula/ansible.rb:525]: No need to move arround recourses

**Review by laserscout — CHANGES_REQUESTED** [review]
Also please have a better look at [these instructions](https://github.com/Homebrew/homebrew-core/pull/50504#issuecomment-592743222) for updating all the rest of the python resources in the formula

**Comment by okket** [comment]
Yes, sorry, I was careless. I updated the pull request and it shrank considerably. Only the main packages seems to have changed, even though I ran through the whole set of instructions.

Thanks for the quick review.

**Review by laserscout — CHANGES_REQUESTED** [review]
There are definitely updates in the reassures.
Random example: `boto3 v1.16.51` is referenced, but the latest version is [boto3 v1.16.60](https://pypi.org/project/boto3/1.16.60/).
Give it another shot :)

**Comment by okket** [comment]
These dependencies result from the instructions. See also

> We update the resources in this way because we want to identify the correct resource versions that the package(s) in the formula ask for, not just the newest version from PyPI.

**Comment by laserscout** [comment]
But if you look at the instructions, for the boto3 example, boto3 is one of the additional requested packages. So it should have the latest versions.
Just try to re-run the pip installation and the poet in a clean venv.

**Comment by okket** [comment]
I don't have venv installed and don't have time, sorry. I will close this pull request and leave it to somebody else.

**Pull request closed without merging**